### DISCUSSION TITLE
Testing dynamic import

### DISCUSSION THREAD

**Discussion by alibey-10** [ljosberinn/personal-react-boilerplate#863] [Q&A]
How do you handle testing dynamic import for Next.js? I solved this issue with babel plugins before but it comes back again and now looking for a more robust solution.

**Comment by ljosberinn** [comment]
Interesting question, I haven't actually tried yet but would assume you can mock your own implementation of `next/dynamic`. I'll try in the upcoming days!

**Reply by ljosberinn** [reply]
related: https://github.com/vercel/next.js/issues/24566 https://github.com/vercel/next.js/pull/24751
